app-backend/db/src/main/resources/migrations/V73__Add_users_intercom_conversations.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE public.user_intercom_conversations (
+    user_id text NOT NULL references users (id) ON DELETE CASCADE,
+    conversation_id text NOT NULL
+);
+
+CREATE INDEX IF NOT EXISTS user_intercom_conversations_user_id_idx ON public.user_intercom_conversations USING btree (user_id);
+CREATE INDEX IF NOT EXISTS user_intercom_conversations_conversation_id_idx ON public.user_intercom_conversations USING btree (conversation_id);
```

Comment:
Do we do lookups by both IDs? Also, I'm curious about whether there are other constraints that we know about, like, is `user_id` unique?

Reply:
Yes! Looks like we may apply the `UNIQUE ` constraint on `user_id`: https://www.postgresqltutorial.com/postgresql-unique-constraint/